poserrank/static/javascript/autocomplete.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+function Autobox(elem, apiSource) {
```

Comment:
What the hell is this function capitalization

Reply:
Why don't you make this into a class? Not a function

Reply:
Lol whoops I didn't realize that was a thing.  Fixing...